plugins/UserCountry/API.php

```diff
@@ -75,6 +95,27 @@ public function getRegion($idSite, $period, $date, $segment = false)
         $separator = Archiver::LOCATION_SEPARATOR;
         $unk = Visit::UNKNOWN_CODE;
 
+        // show visits tracked as Tibet as region of China
+        $dataTables = [$dataTable];
+
+        if ($dataTable instanceof DataTable\Map) {
+            $dataTables = $dataTable->getDataTables();
+        }
+
+        foreach ($dataTables as $dt) {
+            if ($dt->getRowFromLabel('1|ti')) {
+                $dt->filter('GroupBy', array(
+                    'label',
+                    function ($label) {
+                        if ($label == '1|ti') {
+                            return 'cn';
```

Comment:
@sgiehl Thanks for the merge. Should the return value be ```14|cn```?

Reply:
sure. I'll fix that typo